cmd/kubeless/list_test.go

```diff
@@ -59,6 +61,19 @@ func TestList(t *testing.T) {
 					Type:     "ftype",
 					Topic:    "ftopic",
 					Deps:     "fdeps",
+					Template: v1.PodTemplateSpec{
+						Spec: v1.PodSpec{
+							Containers: []v1.Container{
+								{
+									Env: []v1.EnvVar{},
+									Resources: v1.ResourceRequirements{
+										Limits:   map[v1.ResourceName]resource.Quantity{},
+										Requests: map[v1.ResourceName]resource.Quantity{},
```

Comment:
These are all empty, so the zero value should be sufficient.  Can you just remove all these empty initialisers? (and same in "btype" testcase below)